pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -290,6 +290,10 @@ def callback_metrics(self) -> Dict[str, _METRIC]:
             self._callback_metrics.update(metrics)
         return self._callback_metrics
 
+    @callback_metrics.setter
+    def callback_metrics(self, val: dict) -> None:
+        self._callback_metrics = val
+
```

Comment:
Shouldn't need this, you can update the dictionary instead

Reply:
+1 same for Trainer.callback_metrics who forwards here.